common/data_refinery_common/models/models.py

```diff
@@ -306,13 +306,11 @@ def __str__(self):
     # Managers
     objects = models.Manager()
     public_objects = PublicObjectsManager()
+    commands = ArrayField(models.TextField())
```

Comment:
This should be spatially separated from the Manger objects here, which perform a different function